src/main/scala/io/iohk/ethereum/network/p2p/Message.scala

```diff
@@ -26,19 +26,30 @@ object Message {
     case (_, Status.code) => rlp.decode(payload)(Status.rlpEndDec)
     case (_, Transactions.code) => rlp.decode(payload)(Transactions.rlpEndDec)
 
-    case (PV61, pv61.NewBlockHashes.code) => rlp.decode(payload)(pv61.NewBlockHashes.rlpEndDec)
-    case (PV61, pv61.BlockHashesFromNumber.code) => rlp.decode(payload)(pv61.BlockHashesFromNumber.rlpEndDec)
+    case (PV61, t) => handlePV61(t, payload)
 
     case (PV62 | PV63, pv62.NewBlockHashes.code) => rlp.decode(payload)(pv62.NewBlockHashes.rlpEndDec)
     case (PV62 | PV63, pv62.GetBlockHeaders.code) => rlp.decode(payload)(pv62.GetBlockHeaders.rlpEndDec)
     case (PV62 | PV63, pv62.BlockHeaders.code) => rlp.decode(payload)(pv62.BlockHeaders.rlpEndDec)
     case (PV62 | PV63, pv62.GetBlockBodies.code) => rlp.decode(payload)(pv62.GetBlockBodies.rlpEndDec)
     case (PV62 | PV63, pv62.BlockBodies.code) => rlp.decode(payload)(pv62.BlockBodies.rlpEndDec)
```

Comment:
but there is no message specific to PV62, they are shared between PV62 and PV63 so what that method should be called `handlePV62andPartOfPV63` ? for me it is better to leave this messages unnamed in decode method